tools/log-collector/log_collector.go

```diff
@@ -114,6 +114,7 @@ func (l *LogCollector) getResourceObjects(resourcePath string, resource *apiv1.A
 			log.Warnf("%s", err.Error())
 			return objects, nil
 		}
+		// TODO() We will add this feature in future of --continue-on-error type flag
```

Comment:
Does not explain reason why err is ignored now?

Reply:
resolved